docs/src/reservations.md

```diff
@@ -0,0 +1,8 @@
+# Reservations
+
```

Comment:
Should probably have some flavour text here explaining what reservations are and why anyone would use them.

Reply:
Please checkout the documentation.